data/lifestyle.ts

```diff
@@ -217,6 +217,81 @@ let PARTIES = [{
   "value": 10,
 }];
 
+let PROGRAMMING_LANGUAGE = [{
+  "name": "Brainf*ck",
```

Comment:
wow xD was this troll

Reply:
I don't think so lol. Seems to be a thing: https://en.wikipedia.org/wiki/Brainfuck

Reply:
ye i've heard of brainfk; i meant the people who'd actually put this in haha